Code/Tools/SceneAPI/SceneBuilder/Importers/AssImpTransformImporter.cpp

```diff
@@ -56,12 +56,6 @@ namespace AZ
 
```

Comment:
Can you adjust the version on line 42 of this file? That will trigger all FBX files to reprocess. Otherwise people will only see these changes if they manually force a reprocess.